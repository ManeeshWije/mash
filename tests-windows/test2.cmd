printsomething.exe > test4.output
numberlines.exe < test4.output
