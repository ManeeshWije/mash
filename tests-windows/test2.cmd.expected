  0 : Hi there, this is a test
  1 : line two
  2 : line three
  3 : Hi there, this is another test
  4 : Goodbye
